Created Risk Assesment appendix. (#238)
ITU-DevOps2026/ITU-minitwit: report/sections/02-4-security.md

Trivy is run in its own workflow, which scans for misconfigurations in all Dockerfiles present in the repository. It then builds the images and scans for vulnerabilities in those. If any misconfigurations or vulnerabilities are found, it will fail the workflow.
CodeQL runs in its own workflow, which scans the codebase by running a set of queries to find common vulnerabilities in the code base. 
Both workflows runs when a pull request to main is created, on push to main, and each week on Monday. 
Running both workflows on a schedule, rather than just on PR and push is relevant, since new vulnerabilities can be discovered any time, and therefore code that was thought to be secure when the code was pushed, may be discovered to be insecure. The weekly workflow helps ensure that we become aware of newly discovered vulnerabilities.

> In addition to the things discussed in this chapter, we decided to make a Security Assesment of our current system, to determine where we should put our effort, in terms of security, if we had more time to do this project. This can be found in the appendix, in the section [Risk Assesment](#risk-assesment) .

Reviewer: ```suggestion
In addition to the things discussed in this chapter, we also did a Security Assesment of our current system, to determine where we should put our effort, in terms of security, if we had more time to do this project. This can be found in the appendix, in the section [Risk Assesment](#risk-assesment) .
```
